### DISCUSSION TITLE
Browsh over Tor?

### DISCUSSION THREAD

**Discussion by ghost** [browsh-org/browsh#486] [Q&A]
Hi folks!
 I've been playing around with Browsh and tried to get it to use Tor. I edited the config file putting in the needed proxy settings, but the Torproject checker says it's not using Tor. (Yes Tor is running)
 Any one had any experience with this?

Thanks!

**Comment by tariqshaz** [comment]
The Hidden Wiki is a directory of websites on the Tor network, also known as the “dark web” or “deep web.”
The Hidden Wiki is used by a diverse range of individuals, including:
Privacy-conscious users: People who value their privacy and want to access information or services without being tracked or monitored may use the Hidden Wiki to find resources on the Tor network.
Researchers and journalists: Professionals researching topics related to the dark web or conducting investigations may use the Hidden Wiki to find relevant hidden services or communities.
it can be accessed via TOR Browser using this link
http://zqktlwiuavvvqqt4ybvgvi7tyo4hjl5xgfuvpdf6otjiycgwqbym2qad.onion/
